src/Engine/RadiumEngine.hpp

```diff
@@ -185,6 +185,96 @@ class RA_ENGINE_API RadiumEngine
      */
     void popFboAndViewport();
 
+    /// \name Time Management.
+    /// {
+
+    /**
+     * \brief Toggles Real-time (on) or Constant (off) time flow.
+     *
+     * In Real-time mode, time flows according to the actually elapsed time
+     * between consecutive frames, while in constant mode time flows according
+     * to a fixed time step.
+     */
+    void setRealTime( bool realTime );
+
+    /**
+     * \brief Indicates if Real-time time flow is on (Constant-time is off) .
+     * \see isConstantTime
+     */
+    bool isRealTime() const;
+
+    /**
+     * \brief Indicates if Constant-time time flow is on (Real-time is off) .
+     * \see isRealTime
+     */
+    bool isConstantTime() const;
+
+    /**
+     * \brief Sets the time delta between two frames for Constant-time time flow.
+     */
+    void setConstantTimeStep( Scalar dt );
```

Comment:
My point here is that you are not sure, when you call this function, that ConstantTime flow is enabled.
So I propose to add a boolean to force enabling contant time flow while setting the time step, to avoid that the user need to call two functions successively.

Reply:
hmm, so would you set the default value to `true` or `false`?
And why returning the activation state?

Reply:
```c++
bool setConstantTimeStep( Scalar dt, bool forceConstantTime = false );
```
permet de garder un code lisible dans les cas généraux (et sans effet de bord) ou l'on souhaite simplement changer le timestep, ie:
```c++
foo.setConstantTimeStep( .1_ra );
```
Cela peut par être par exemple connecté à un wdget qui permet à l'utilisateur de changer le timestep.

Autre usage, un plugin de simulation qui veut lancer une simulation avec ses propres paramètres, et qui souhaite donc activer le mode constant:
```c++
foo.setConstantTimeStep( .1_ra, true ); // will always return true
```

Concernant la valeur de retour, elle peut être utile pour informer l'utilisateur de certains comportements:
```c++
if( ! foo.setConstantTimeStep( .1_ra ) )
   // log: warning: constantTimeStep set but RealTime mode is enabled
```

It is just a matter of design (very minor point) and I will not hang on this if others don't like it,.

Reply:
ok